api/src/accounts.rs

```diff
@@ -164,50 +165,88 @@ impl Account {
             .map_err(BasicErrorWith404::internal)?;
         let account_data: AccountData = account_resource.into();
 
-        BasicResponse::try_from_rust_value((
-            account_data,
-            &self.latest_ledger_info,
-            BasicResponseStatus::Ok,
-            accept_type,
-        ))
+        match accept_type {
+            AcceptType::Json => BasicResponse::try_from_json((
+                account_data,
+                &self.latest_ledger_info,
+                BasicResponseStatus::Ok,
+            )),
+            AcceptType::Bcs => BasicResponse::try_from_encoded((
```

Comment:
will this break @jjleng's BCS API?
if so we should hold off on the wallet using this

Reply:
I've got to look at what it uses.  This is exactly why I said earlier we shouldn't allow responses on the BCS API unless it's been properly completed.